yb-voyager/cmd/exportDataFromTarget.go

```diff
@@ -80,12 +95,14 @@ func initSourceConfFromTargetConf() error {
 	} else {
 		source.Schema = targetConf.Schema
 	}
-	if source.SSLMode == "" {
-		source.SSLMode = targetConf.SSLMode
+	if targetConf.SSLCertPath != "" || targetConf.SSLKey != "" {
```

Comment:
if sslmode is "disable", we shouldn't even invoke this prompt

Reply:
Done